### PR TITLE
Governance 2 (Part 1, Non-Controversial Changes)

### PR DESCRIPTION
**Pull request by shawntabrizi**
Breaking apart some of the changes from: https://github.com/paritytech/polkadot/pull/5205

These should be all non-controversial changes, and refactors the exact governance changes from master, making it easier to audit the changes from the #5205.

### CODE REVIEW

File: runtime/kusama/src/governance/old.rs [hunk 1]
```diff
@@ -0,0 +1,172 @@
+// Copyright 2022 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
```
Comment (shawntabrizi) [inline runtime/kusama/src/governance/old.rs:4]: This file is exactly copy and paste from the existing runtime configuration.

**Review by ggwpez — APPROVED** [review]

File: runtime/kusama/src/governance/mod.rs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot. If not, see <http://www.gnu.org/licenses/>.
+
+//! Governance configurations for the Kusama runtime.
+use super::*;
```
Comment (bkchr) [inline runtime/kusama/src/governance/mod.rs:18]:
```suggestion
```

File: runtime/kusama/src/governance/old.rs [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot. If not, see <http://www.gnu.org/licenses/>.
+
+//! Old governance configurations for the Kusama runtime.
+
+use super::*;
```
Comment (bkchr) [inline runtime/kusama/src/governance/old.rs:19]:
```suggestion
use crate::*;
```

File: xcm/xcm-builder/src/origin_conversion.rs [hunk 4]
```diff
@@ -253,8 +253,8 @@ where
 	}
 
 	#[cfg(feature = "runtime-benchmarks")]
-	fn successful_origin() -> Origin {
-		Origin::root()
+	fn try_successful_origin() -> Result<Origin, ()> {
```
Comment (bkchr) [inline xcm/xcm-builder/src/origin_conversion.rs:256]: Was this renamed in substrate?

File: runtime/common/src/slots/mod.rs [hunk 5]
```diff
@@ -1108,15 +1109,17 @@ mod benchmarking {
...
+				Slots::<T>::force_lease(origin, para, leaser, amount, period_begin, period_count)?;
 			}
 
 			for i in 0 .. max_people {
 				let leaser = account("lease_deposit", i, 0);
 				assert_eq!(T::Currency::reserved_balance(&leaser), T::Currency::minimum_balance());
 			}
 
-		}: _(RawOrigin::Root, para)
+			let origin = T::ForceOrigin::successful_origin();
```
Comment (bkchr) [inline runtime/common/src/slots/mod.rs:1121]: Here it is again without try?

**Review by bkchr — APPROVED** [review]

File: runtime/common/src/slots/mod.rs (hunk 5 above)
Comment (shawntabrizi) [reply runtime/common/src/slots/mod.rs:1121]: There is `try_successful_origin` and `successful_origin`, which is automatically implemented as `try_successful_origin().unwrap()`.

In this case, we need the successful origin

File: xcm/xcm-builder/src/origin_conversion.rs (hunk 4 above)
Comment (shawntabrizi) [reply xcm/xcm-builder/src/origin_conversion.rs:256]: https://github.com/paritytech/substrate/pull/11558

**Review by kianenigma — APPROVED** [review]

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**